package.json

```diff
@@ -23,11 +23,11 @@
         "eslint-plugin-unicorn": "47.0.0",
         "prettier": "2.8.8",
         "rimraf": "5.0.1",
-        "typedoc": "0.24.7",
+        "typedoc": "0.25.0",
         "typescript": "5.0.4"
     },
     "peerDependencies": {
-        "typedoc": "0.24.x"
+        "typedoc": "^0.24.x"

```

Comment:
Hi. Thanks for your PR.
This doesn't modify the allowed versions: https://jubianchi.github.io/semver-check/#/^0.24.x/0.25.0
I will check if the latest major verison is working properly with the plugin and update it accordingly.

Reply:
Oops, you are very right! I have fixed it and also added a comment in the code to explain a change I made.
If you prefer to close this and open a PR yourself though that's totally ok - feel free to go ahead :)